src/sentry/api/endpoints/data_export_details.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+# from sentry import features
+from sentry.api.base import Endpoint
+from sentry.api.bases.incident import IncidentPermission
+from sentry.api.serializers import serialize
+from sentry.models import ExportedData
+
+
+class DataExportDetailsEndpoint(Endpoint):
+    permission_classes = (IncidentPermission,)
```

Comment:
`IncidentPermission` is probably not what you want here. Who has access to these endpoints?